docs/querying/basics.md

```diff
@@ -335,7 +337,7 @@ independently of the actual present time series data. This is mainly to support
 cases like aggregation (`sum`, `avg`, and so on), where multiple aggregated
 time series do not precisely align in time. Because of their independence,
 Prometheus needs to assign a value at those timestamps for each relevant time
-series. It does so by taking the newest sample before this timestamp within the lookback period.
+series. It does so by taking the newest sample before this timestamp within the left-open lookback period,
```

Comment:
I'm not sure if "left-open" will be understood in this context. How about the following:

```suggestion
series. It does so by taking the newest sample that is less than the lookback period ago.
```